src/Lykke.Service.BlockchainApi.Sdk/ServiceCollectionExtensions.cs

```diff
@@ -65,6 +66,17 @@ public static class ServiceCollectionExtensions
             return services;
         }
 
+        static IServiceCollection AddChaos(this IServiceCollection services,
+            ChaosSettings chaosSettings)
+        {
+            if (chaosSettings != null)
+            {
+                services.AddSingleton<IChaosKitty>(sp => new ChaosKitty(chaosSettings.StateOfChaos));
```

Comment:
Not necessary to fix, just note for the future - there is the extension method, that you could use to register IChaosKitty: https://github.com/LykkeCity/CommonDotNetLibraries/blob/master/src/Lykke.Common/Chaos/ContainerBuilderExtensions.cs#L17 if this method is used, then you don't need to use ```?.Meow()``` everywhere.

Reply:
I know about this extension. But Autofac is not used here and ContainerBuilder is not available at the time of registration of services (or it requires another registration approach - through Autofac modules).